modules/reactor/doc/content/source/meshgenerators/AzimuthalBlockSplitGenerator.md

```diff
@@ -15,6 +15,8 @@ Multiple radial blocks, which are defined by [!param](/Mesh/AzimuthalBlockSplitG
 
 As moving nodes in the azimuthal direction changes the volumes of the circular blocks, the volume preservation radius correction is made if [!param](/Mesh/AzimuthalBlockSplitGenerator/preserve_volumes) is set as true.
 
+If the [!param](/Mesh/AzimuthalBlockSplitGenerator/input) mesh is generated by [`PolygonConcentricCircleMeshGenerator`](/PolygonConcentricCircleMeshGenerator.md) with [!param](/Mesh/PolygonConcentricCircleMeshGenerator/quad_center_elements) set as true, and if the block that contains these quadrilateral central elements is selected in [!param](/Mesh/AzimuthalBlockSplitGenerator/old_blocks), an error message is issued because no nodes in that block should be moved.
```

Comment:
```suggestion
If the [!param](/Mesh/AzimuthalBlockSplitGenerator/input) mesh is generated by [`PolygonConcentricCircleMeshGenerator`](/PolygonConcentricCircleMeshGenerator.md) with [!param](/Mesh/PolygonConcentricCircleMeshGenerator/quad_center_elements) set as true, and if the block that contains these quadrilateral central elements is selected in [!param](/Mesh/AzimuthalBlockSplitGenerator/old_blocks), an error message is issued because no nodes in that block should be moved. The central pattern of quad elements is not designed to be arbitrarily modified to align its nodes with azimuthal sectors.
```